cmd/thanos/query_frontend.go

```diff
@@ -199,6 +199,8 @@ func runQueryFrontend(
 		instr := func(f http.HandlerFunc) http.HandlerFunc {
 			hf := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 				name := "query-frontend"
+				// Set CORS to frontend client
+				w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
Can we do `api.SetCORS` instead? (:  And no need for comment :hugs: 

Thanks!

Reply:
Ref: https://github.com/bwplotka/thanos/blob/987b74670cd5ab8bab15cd35fccc4c85dd36ce9e/pkg/api/api.go#L118

Reply:
@bwplotka I initially did it like you said but I thought that maybe I shouldn't set  more that what I actually needed. Let me change that again.